Stylish/JSONStylesheet.swift

```diff
@@ -51,17 +51,32 @@ public class JSONStylesheet: Stylesheet {
     /// - Throws: An NSError with a description of what went wrong
     public init(data: Data, usingPropertyStylerTypes propertyStylerTypes: [AnyPropertyStylerType.Type] = Stylish.builtInPropertyStylerTypes, ignoringUnrecognizedStyleProperties: Bool = false) throws {
         let propertyStylerTypes = Dictionary(propertyStylerTypes.map{(($0.wrapped as! AnyPropertyStylerTypeWrapper).propertyKey, $0.wrapped as! AnyPropertyStylerTypeWrapper)}, uniquingKeysWith: { $1 })
-        guard let json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any], let styles = json["styles"] as? [[String: Any]] else {
+        guard let json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any],
+            let styles = json["styles"] as? [[String: Any]],
+            let constants = json["constants"] as? [String: Any] else {
```

Comment:
Requiring the "constants" key here (otherwise dropping into the error throwing exit) will break existing JSON stylesheets in existing projects.  How about making `constants` an optional dictionary outside of this guard, and then unwrapping it down around line 70 below?

Reply:
makes total sense! will apply that now

Reply:
Changes applied